### PR TITLE
Ignore php.ini and .php-version

### PR DESCRIPTION
**Pull request by vvasiloi**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets |
| Related issues/PRs |
| License | MIT
| Documentation PR |

#### What's in this PR?

Ignore php.ini and .php-version which are used when working with Symfony CLI.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -57,6 +57,14 @@ phpunit.xml
 jackrabbit-standalone*
 jackrabbit/*
 
+# Project level ini settings, primarily for Symfony CLI
+# https://symfony.com/doc/current/setup/symfony_server.html#overriding-php-config-options-per-project
+php.ini
```
Comment (alexander-schranz) [inline .gitignore]: Interesting didn't know symfony supports this :)

File: .gitignore [hunk 2]
```diff
@@ -57,6 +57,14 @@ phpunit.xml
...
+
+# Project level PHP version to run with Symfony CLI
+# https://symfony.com/doc/current/setup/symfony_server.html#selecting-a-different-php-version
+.php-version
+
```
Comment (alexander-schranz) [inline .gitignore]:
```suggestion
# Symfony CLI
# https://symfony.com/doc/current/setup/symfony_server.html#different-php-settings-per-project
/php.ini
/.php-version
```

Would just combine them like we do it for other section and only exclude the root files for project

File: .gitignore (hunk 1 above)
Comment (vvasiloi) [reply .gitignore]: Quite handy. I use it mostly for xdebug settings, memory limit, max execution time and error reporting.

**Comment by alexander-schranz** [comment]
@vvasiloi looks good just a small change request. And can you target the `2.4` branch for this and rebase the branch:

```bash
git checkout patch-1 # checkout your branch

git remote add sulu git@github.com:sulu/sulu.git
git fetch sulu

git rebase -i origin/2.5 --onto origin/2.4

git push yourremote patch-1 --force # force push required when rebasing
```

File: .gitignore (hunk 2 above)
Comment (vvasiloi) [reply .gitignore]: Sure. The reason I didn't restrict them to root is to allow using different settings per bundle, but that's probably unnecessary.

**Comment by vvasiloi** [comment]
@alexander-schranz done!

**Comment by alexander-schranz** [comment]
@vvasiloi can you repush your commit with a new hash:

```
git commit --amend --date="now"
git push yourremote patch-1 --force
```

Github actions are getting confused when target branch is changed without something commited after it.

**Comment by vvasiloi** [comment]
@alexander-schranz done

**Comment by alexander-schranz** [comment]
@vvasiloi Thx for the improvement if you stumble over other things which can make contributing easier let us know :)

**Pull request merged**